packages/model/src/user.ts

```diff
@@ -81,6 +81,11 @@ export interface OrcidWork {
   lastModifiedDate: string;
 }
 
+export interface UserLab {
```

Comment:
to be fair i dont think this needs to be a user-specific type.

Labs unlike other entities are not being returned on their own, and so there is no direct Response type definition inside /entities - I would still probably create a file and have a basic Lab definition there that we might use for teams etc.